README.md

```diff
@@ -458,7 +458,7 @@ module Articles
   class Show
     include Lotus::Action
 
-    handle_exception StandardError => :handle_standard_error
+    handle_exception Lotus::Controller::Error => :handle_controller_error
```

Comment:
@karimmtarek This is unrelated to the scope of your change, please revert it. :smile: